doc/specs/vulkan/chapters/initialization.txt

```diff
@@ -54,12 +54,12 @@ be cast to the type of the command being queried.
 
 1::
 The returned function pointer must: only be called with a dispatchable object
-(the first parameter) that is a child of pname:instance.
+(the first parameter) that is sname:VkInstance or a child of pname:instance.
```

Comment:
This is mixing types and objects. To my understanding, it must be the `pname:instance` (that is the same object used by GIPA)